docs/index.html

```diff
@@ -0,0 +1,93 @@
+            <div>
+                <i class="fa-brands fa-facebook-f"></i>
+                <i class="fa-brands fa-twitter"></i>
+                <a href="#">English</a>
+                <a href="#">My page</a>
+                <a href="#">Logout</a>
+            </div>
+        </nav>
+        <nav class="nav-bar nav-bar-2 drop-shadow">
+            <div>
+                <a href="/get-inspired/index.html"><img src="./sprites/22-removebg-preview.png" alt="Get inspired logo"></a>
+            </div>
+            <div class="no-margin">
+                <a  href="/get-inspired/templates/about.html">About</a>
+                <a href="#top">Program</a>
+                <a href="#">Join</a>
+                <a href="#partner">Sponsor</a>
+                <a href="#">News</a>
+                <button type="button">CC Campaign</button>
+            </div>
+        </nav>
+        <header class="header general-margin">
+            <h3 class="h3 overflow">"Hello developers and code lovers!"</h3>
+            <h1 class="h1 overflow">GET INSPIRED BY GREAT SOFTWARE DEVELOPERS</h1>
+        </header>
+        <p class="general-margin center-border  normal-font">
+            A compilation of resources about programmers and their acoomplishments in their career. If you are 
+            looking for successful people to look up to, search for inspiration, or simple
+            admire a good software career path and work.
+        </p>
+        <div class="general-margin">
+            <h2 class="bold-hash overflow">#software #coding #development</h2>
+            <h4 class="bold-hash-2">@ Lovers Of Software development, Art of coding corner</h4>
+        </div>
+    </section>
+    <!-- Section 2 -->
+    <section class="section-2 overflow">
+        <h2 id="top" class="align-self-center section-title">Topics</h2>
+        <div class="guide-bar"></div>
+        <div class="art-hold">
+            <article class="article general-margin"><img class="article-img" src="sprites/emb.png" alt="Embedded systems logo"><h3 class="text-overflow topic-style">Embedded systems</h3>
+            </article>
+            <article class="article general-margin"><img class="article-img" src="sprites/Protocol.png" alt="Protocols logo"><h3 class="text-overflow topic-style">Protocols</h3>
+            </article>
+            <article class="article general-margin"><img class="article-img" src="sprites/OS.png" alt="Operating Systems logo"><h3 class="text-overflow topic-style">Operating Systems</h3>
+            </article>
+            <article class="article general-margin"><img class="article-img" src="sprites/progra.png" alt="Programming languagues logo"><h3 class="text-overflow topic-style">Programming languages</h3>
+            </article>
+            <article class="article general-margin"><img class="article-img" src="sprites/algo.png" alt="Algorithms logo"><h3 class="text-overflow topic-style">Algorithms</h3>
+            </article>
+        </div>
```

Comment:
I hope you're doing well 😀, In this section, you are supposed to add white text in order to match the given design 

Given Design: 
![image](https://user-images.githubusercontent.com/87448628/163600086-89e140c6-6209-40a8-9472-ddd07cd9171b.png)

Your Design: 
![image](https://user-images.githubusercontent.com/87448628/163600130-8918b0d1-6634-4279-aaf0-7551e093ded3.png)

Kindly Fix this issue ✔👍.